README.md

```diff
@@ -35,12 +63,23 @@ Download the minified library from the [CDN](https://unpkg.com/@hyperapp/logger)
 <script src="https://unpkg.com/@hyperapp/logger"></script>
 ```
 
-You can find the library in `window.logger`.
+You can find the library in `window.hyperappLogger`.
+
+```js
+const { withLogger } = hyperappLogger
+```
 
 ## Usage
 
+Compose the `withLogger` function with your `app` before calling it with the usual arguments.
+
 ```js
-logger(options)(app)(state, actions, view, document.body)
+import { withLogger } from "@hyperapp/logger"
+
+withLogger(options)(app)(state, actions, view, document.body)
```

Comment:
Maybe make focus on simplicity - show "no options" example first and later "if you need options you can use this signature"

Reply:
I switched the order now, what do you think?